package.json

```diff
@@ -18,13 +18,13 @@
     "eslint-config-prettier": "^9.1.0",
     "eslint-plugin-import": "^2.29.1",
     "eslint-plugin-jsdoc": "^48.2.5",
-    "lerna": "^5.6.2",
+    "lerna": "^8.1.8",
     "lerna-update-wizard": "^0.17.5",
     "prettier": "^3.2.5",
     "ts-api-utils": "~1.0.1",
     "type-coverage": "^2.26.3",
-    "typedoc": "^0.25.13",
-    "typedoc-plugin-markdown": "^3.17.1",
+    "typedoc": "^0.26.6",
+    "typedoc-plugin-markdown": "^4.2.5",
```

Comment:
heads up @Jovonni @amessbee since this is part of the docs.agoric.com build pipeline. I don't know if we have CI integration to catch if this breaks anything. I'm approving since it is low impact if it does break temporarily